util.go

```diff
@@ -50,11 +57,21 @@ func writeLines(name string, lines []string, truncate bool) error {
 		if err != nil {
 			return fmt.Errorf("cannot open file for appending: %w", err)
 		}
-		defer file.Close()
+
+		// When it is not possible to close file, then return error.
+		// This is very important in situation, when system call close()
+		// returns EIO. It means that previously-uncommitted write(2)
+		// encountered an input/output error, and we lost data.
+		defer func() {
+			closeErr := file.Close()
+			if closeErr != nil {
+				err = closeErr
+			}
+		}()
```

Comment:
Again, this is fine as it is. I'd like to ask if you could consider this DRY-er suggestions:

```
        defer func() {
                err = file.Close()
        }()

```